problem-21.c

```diff
@@ -23,11 +23,11 @@ int main()
         scanf("%f", &percentages[i]);
     }
 
-    printf("Name   Roll no   %c\n", 37);
-
     for (int i = 0; i < 5; i++)
     {
-        printf("%s\t %d      %.1f\n", names[i], roll_numbers[i], percentages[i]);

```

Comment:
Is it requirement to show the result in the forms of coloums or we can use any way of representation?
cc: @mehmoodak